### PR TITLE
New User

### PR DESCRIPTION
**Pull request by getonlineamit**

### CODE REVIEW

File: images/users/snapdeal.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...15.316 8.56 0 12.286 4.868 12.286 14.606v25.762c0 2.13 1.71 3.854 3.826 3.854h.202c2.115 0 3.826-1.723 3.826-3.854V47.67c0-14.706-5.64-21.705-18.327-21.705-6.95 0-12.185 2.638-15.51 7.405v-2.23c0-2.03-1.71-3.755-3.724-3.755h-.2c-2.115 0-3.828 1.725-3.828 3.754v42.496c0 2.13 1.713 3.854 3.828 3.854h.2c2.115 0 3.827-1.723 3.827-3.854M474.17 78.2c2.115 0 3.927-1.52 3.927-3.752 0-4.057-4.733-3.45-4.733-10.244V4.057c0-2.13-1.712-3.854-3.827-3.854h-.2c-2.116 0-3.828 1.725-3.828 3.854v62.277c0 5.984 4.33 11.867 8.66 11.867" fill="#333"/><g transform="translate(0 4.91)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M21.718 90.23c1.108 1.927 2.316 2.637 4.53 2.637h41.29c2.215 0 3.423-.61 4.833-2.435l31.22-40.57c.703-.914-.102-2.435-1.21-2.435H52.33c-2.214 0-3.423-.71-4.53-2.636L22.624.974c-.603-1.014-2.215-.81-2.618.203L.57 48.543c-.905 2.13-.704 3.55.404 5.477l20.744 36.21z" fill="#E40046" mask="url(#b)"/></g><path d="M73.68 42.803c1.108 1.927 2.316 2.637 4.53 2.637h24.17c1.208 0 2.115-1.522 1.51-2.637L80.83 2.638C79.723.71 78.514 0 76.3 0H30.175c-1.207 0-2.113 1.52-1.51 2.638L40.65 23.532c1.108 1.926 2.316 2.636 4.532 2.636h19.032l9.466 16.635z" fill="#E40046"/></g></svg>
```
Comment (xtuc) [inline images/users/snapdeal.svg]: As mentioned at the bottom of the user page (http://babeljs.io/users), could you please submit a 500x200 svg.

Comment (xtuc) [inline images/users/snapdeal.svg]: As mentioned at the bottom of the user page (http://babeljs.io/users), could you please submit a 500x200 svg.

**Comment by getonlineamit** [comment]
@xtuc I have made changes as requested.

File: images/users/snapdeal.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                        </g>
+                        <path d="M73.6799824,42.8025355 C74.7875293,44.7300879 75.9962735,45.4401792 78.2113674,45.4401792 L102.379504,45.4401792 C103.588248,45.4401792 104.494525,43.9180696 103.890715,42.8025355 L80.8301256,2.63764368 C79.7225786,0.710091278 78.5138345,0 76.2987406,0 L30.1764378,0 C28.9688181,0 28.0625411,1.52097701 28.6663509,2.63764368 L40.6503462,23.5315416 C41.7578932,25.4579615 42.9655129,26.1680527 45.1817312,26.1680527 L64.2135481,26.1680527 L73.6799824,42.8025355 Z" id="Fill-20" fill="#E40046"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (xtuc) [inline images/users/snapdeal.svg]: Could you please run [svgo](https://jakearchibald.github.io/svgomg/) on your SVG to optimize it.

**Comment by getonlineamit** [comment]
@xtuc changes done

**Pull request merged**